Frontend/adminhrms.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>CYBER HRMS</title>
+
+
```

Comment:
Few Meta Tags missing

Reply:
Meta tags have now been added.Will commit the changes.